lib/utilities/ember-cli-babel-config-key.js

```diff
@@ -1,12 +0,0 @@
-'use strict';
-
-module.exports = function emberCLIBabelConfigKey(emberCLIBabelInstance) {
-  // future versions of ember-cli-babel will be moving the location for its
-  // own configuration options out of `babel` and will be issuing a deprecation
-  // if used in the older way
-  //
-  // see: https://github.com/babel/ember-cli-babel/pull/105
-  let emberCLIBabelConfigKey = (emberCLIBabelInstance && emberCLIBabelInstance.configKey) || 'babel';
```

Comment:
As long as `ember-cli` uses this fallback, we can't remove the `configKey` property in `ember-cli-babel`. Even after this change, we have to wait a while before we can remove it in `ember-cli-babel`.
Otherwise, people using an older `ember-cli` version might run into to trouble.